app/views/practices/introduction.html.erb

```diff
@@ -0,0 +1,192 @@
+<% provide :head_tags do %>
```

Comment:
do we need to get rid of the overview page since they are now the same thing?

Reply:
ooo I see... this is what brad added (i'm assuming)... so maybe we want to hide the introduction page?

Reply:
delete from this PR. probably in the route, too